github-profiles/haboufaker-8509/index.html

```diff
@@ -3,41 +3,9 @@
     <head>
         <meta charset="UTF-8">
         <title>Hi there &#x1f44b;&comma; my name is Hisam Abou Faker</title>
-        <style>
-/* From extension vscode.github */
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-.vscode-dark img[src$=\#gh-light-mode-only],
-.vscode-light img[src$=\#gh-dark-mode-only] {
-	display: none;
-}
-
-</style>
-        
+        <link rel='stylesheet' href='styles.css'/>
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
-<style>
-            body {
-                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
-                font-size: 14px;
-                line-height: 1.6;
-            }
-        </style>
-        <style>
-.task-list-item {
-    list-style-type: none;
-}
-
-.task-list-item-checkbox {
-    margin-left: -20px;
-    vertical-align: middle;
-    pointer-events: none;
-}
-</style>
-        
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
     </head>
     <body class="vscode-body vscode-light">
         <h1 id="hi-there--my-name-is-hisam-abou-faker">Hi there 👋, my name is Hisam Abou Faker</h1>
```

Comment:
Hola @haboufaker  mi sugerencia para los id's o clases de tus etiquetas es que posean un nombre más corto y que dé a entender qué función cubrirá o propósito tendrá en tu maqueta, por ejemplo, para este caso para este `H1` el id podría cambiar de `hi-there--my-name-is-hisam-abou-faker` a `greetings-title`. Otro punto a tomar en cuenta es no colocar más de un guión seguido en la definición.